ci/packages/raspberry.go

```diff
@@ -100,7 +98,7 @@ func configureRaspbianImage(raspbianImagePath string) error {
 	}
 	defer func() {
 		if err := device.DetachLoop(loopDevice); err != nil {
-			log.Warn(err)
+			log.Warn().Err(err).Msg("")
```

Comment:
This looks a lot more verbose and less performant.. 3 functions instead of 1?

Reply:
Have you tried benchmarking it?

Reply:
I have considered making a facade for it to keep simple log calls simple, but with the added ability to log typed structured fields, it just does not fit, thus we would have use both the facade and zerolog/log. So I decided to simply go with zerolog directly.